src/test/java/com/raisetech/inventoryapi/ProductMapperTest.java

```diff
@@ -97,4 +97,38 @@ class ProductMapperTest {
         assertThat(productMapper.findById(1)).contains(new Product(1, "Shaft"));
     }
 
+    @Test
+    @Sql(
+            scripts = {"classpath:/delete-products.sql", "classpath:/insert-products.sql"},
+            executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD
+    )
+    @Transactional
+    void 削除処理が完了して商品情報が削除されること() {
+        productMapper.deleteProductById(1);
+        List<Product> products = productMapper.findAll();
+        assertThat(products)
+                .hasSize(1)
+                .contains(
+                        new Product(2, "Washer")
+                );
+    }
+
+    @Test
+    @Sql(
+            scripts = {"classpath:/delete-products.sql", "classpath:/insert-products.sql"},
+            executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD
+    )
+    @Transactional
+    void 削除処理時に存在しないIDを指定した場合空で返すこと() {
```

Comment:
削除処理の結果として空の値が返されるという記述に違和感あります 🚀 
```suggestion
    void 削除処理時に存在しないIDを指定した場合何も削除されないこと() {
```

Reply:
コメントありがとうございます！
確かに、空で返す わけではない事、理解しました。
5b594c17c68013c8a66382c12ff8fcc470cfd9ff